commit 08dadaf3f345f3feceefaacebf5045c72821b449
Author: Simon Richter <Simon.Richter@hogyros.de>
Date:   Tue Apr 2 15:11:01 2013 +0200

    "Implement" clGetExtensionFunctionAddress()
    
    This function can legally return NULL in the current implementation.

Index: beignet-0.0.0+git2013.04.11+e6b503e/src/cl_api.c
===================================================================
--- beignet-0.0.0+git2013.04.11+e6b503e.orig/src/cl_api.c	2013-04-12 08:13:48.000000000 +0200
+++ beignet-0.0.0+git2013.04.11+e6b503e/src/cl_api.c	2013-04-15 18:33:32.788352738 +0200
@@ -1169,7 +1169,7 @@
 void*
 clGetExtensionFunctionAddress(const char *func_name)
 {
-  NOT_IMPLEMENTED;
+  /* No extensions supported at present */
   return NULL;
 }
 
